p/scripts/main.js

```diff
@@ -863,8 +863,11 @@ function init_column_categories() {
 				a.href = '#dropdown-' + id;
 				div.querySelector('.dropdown-target').id = 'dropdown-' + id;
 				div.insertAdjacentHTML('beforeend', template);
-				if (feed_web.length < 1) {
-					div.querySelector('.item.link.website').remove();
+				if (feed_web == '') {
+					const website = div.querySelector('.item.link.website');
```

Comment:
I do not dare using *Optional chaining operator* `?.` https://caniuse.com/?search=%3F. just yet :-)

Reply:
That wouldn't work here anyway. Or in any case not if you meant `feed_web?.length`. `feed_web` would need to exist, but then `feed_web?.lala?.blabla?.albalb` wouldn't fail.

What I wrote would work (`feed_web && feed_web.length`), but sticking to a string is probably clearer. Incidentally I only just noticed you didn't write `===` though.

Reply:
I meant `div.querySelector('.item.link.website')?.remove();`

Reply:
That one should work. :-)